### PR TITLE
Add tracking_disable=True as default context to load CSVs

### LINKED ISSUE #42: lyrics.loader.load_csv - tracking disable & noupdate
load_csv being mostly used to load tons of data, wouldn't it make sens to change the default values?

Usage has shown that we almost use tracking disabled every time.
Plus, data created from CSV is mostly not config and shouldn't be modified update, thus it should also
set the xml ids as `noupdate`. 

We can change the current function default behavior or have an alias that does this small change.

For me there is 2 use case:

- Most common: create new record data csv, data as the user create them, we don't want any module changing them (fake module name does the trick but it's hacky)
- Less common: Modify existing data, can be used for data that already exists in database, you don't care about noupdate because it is already in that state.

### PR DESCRIPTION
**Pull request by grindtildeath**
closes #42

### CODE REVIEW

File: anthem/lyrics/loaders.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def read_csv(data, dialect='excel', encoding='utf-8', **fmtparams):
 
 def load_rows(ctx, model, header, rows):
     if isinstance(model, basestring):
-        model = ctx.env[model]
+        model = ctx.env[model].with_context(tracking_disable=True)
```
Comment (sebalix) [inline anthem/lyrics/loaders.py]: And in the case there a need to enable the tracking, is there a solution to force the value when calling `load_rows`?

Comment (grindtildeath) [reply anthem/lyrics/loaders.py]: Yes, by passing an instance of `models` with the context key defined. I added a commit to cover more use cases.

Comment (yvaucher) [reply anthem/lyrics/loaders.py]: @sebalix prior to this change the only way to define the context was already to pass a RecordSet with a specific context, so for me the current change makes sense.

File: anthem/lyrics/loaders.py [hunk 2]
```diff
@@ -57,7 +57,10 @@ def read_csv(data, dialect='excel', encoding='utf-8', **fmtparams):
...
+    else:
+        if 'tracking_disable' not in model.env.context.keys():
```
Comment (yvaucher) [inline anthem/lyrics/loaders.py]: `.keys()` is not necessary

**Pull request merged**